mlir-sycl/include/mlir/Dialect/SYCL/Transforms/Passes.h

```diff
@@ -21,7 +21,7 @@ namespace mlir {
 namespace sycl {
 
 /// Inline mode to attempt.
-enum InlineMode { AlwaysInline, Simple, Aggressive };
+enum InlineMode { AlwaysInline, Simple, Aggressive, Ludicrous };
```

Comment:
Should InlineMode also impact default `MaxIterationCount`?

Reply:
I'm inclined to keep then 2 separated but I'm open to preferences. Is that your preference?

Reply:
Discussed offline. Decided we want to set the MaxIterationCount depending on the InlineMode (can always revisit this decision later if we need too).